.changes/v2.17.0/502-features.md

```diff
@@ -0,0 +1,4 @@
+* Added new methods `VCDClient.GetProviderVdcByHref`, `VCDClient.GetProviderVdcById`, `VCDClient.GetProviderVdcByName` and `ProviderVdc.Refresh` to retrieve Provider VDCs [GH-502]
```

Comment:
This comment is about PR title. Please add there that it's for read/get functions only. Something like this:
> Add support for Provider VDC **read functions** ...

Reply:
Done!